aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -339,6 +358,12 @@ public class AwsProperties implements Serializable {
    */
   public static final String CLIENT_ASSUME_ROLE_REGION = "client.assume-role.region";
 
+  /**
+   * Used by {@link org.apache.iceberg.aws.AwsClientFactories.DefaultAwsClientFactory}. If set, all
+   * AWS clients except STS client will use * the given region instead of the default region chain.
+   */
+  public static final String AWS_CLIENT_REGION = "client.region";
```

Comment:
similar comment for variable name, should be `clientRegion` instead of `awsRegion`, and `CLIENT_CLIENT_REGION` instead of `AWS_CLIENT_REGION`.

Reply:
Changed as suggested.